### PR TITLE
feat: form组件新增disabled能力

### PR DESCRIPTION
**Pull request by zym19960704**
**What kind of change does this PR introduce?** (check at least one)

-   [ ] Bugfix
-   [x] Feature
-   [ ] Code style update
-   [ ] Refactor
-   [ ] Docs
-   [ ] Build-related changes
-   [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

-   [ ] Yes
-   [x] No

If yes, please describe the impact and migration path for existing applications:

**Related issue (if exists):**

**Other information:**

### CODE REVIEW

File: components/_util/use/useSelect.ts [hunk 1]
```diff
@@ -33,10 +33,17 @@ export default ({
...
+            props.disabled ||
+            (isGroup && group?.props?.disabled) ||
+            isDisabled.value,
     );
     const handleClick = () => {
-        if (props.disabled || (isGroup && group?.props?.disabled)) {
+        if (
+            props.disabled ||
+            (isGroup && group?.props?.disabled) ||
+            isDisabled.value
```
Comment (wanchun) [inline components/_util/use/useSelect.ts:45]: 用disabled.value

Comment (wanchun) [inline components/_util/use/useSelect.ts:45]: 用disabled.value

**Comment by zym19960704** [comment]
您好，您的邮件我已收到。我将尽快给您回复，谢谢您的来件。

File: components/form/formItem.vue [hunk 2]
```diff
@@ -253,6 +254,9 @@ export default defineComponent({
             isError: computed(() => {
                 return validateStatus.value === VALIDATE_STATUS.ERROR;
             }),
+            isDisabled: computed(() => {
```
Comment (wanchun) [inline components/form/formItem.vue:257]: 可以直接： isDisabled: disabled

File: components/upload/trigger.vue [hunk 3]
```diff
@@ -22,14 +22,26 @@
...
+    accept,
+    disabled: uploadDisabled,
+    prefixCls,
+    onUploadFiles,
+    inputRef,
+} = inject(key);
+
+// 表单组件的总体disabled状态
+const { isDisabled } = useFormAdaptor();
+const disabled = ref(uploadDisabled || isDisabled.value);
```
Comment (wanchun) [inline components/upload/trigger.vue:44]: ref不行，要用computed

File: components/upload/uploadDragger.tsx [hunk 4]
```diff
@@ -37,9 +38,13 @@ export default defineComponent({
             isDragger,
             accept,
             multiple,
-            disabled,
+            disabled: uploadDisabled, // 组件本身的disabled 状态
         } = inject(key);
 
+        // 表单组件的总体disabled状态
+        const { isDisabled } = useFormAdaptor();
+        const disabled = ref(uploadDisabled || isDisabled.value);
```
Comment (wanchun) [inline components/upload/uploadDragger.tsx:46]: ref改为computed

**Pull request merged**